SingularityUI/app/components/taskSearch/QueryParam.cjsx

```diff
@@ -0,0 +1,14 @@
+QueryParam = React.createClass
```

Comment:
This is a helpful component - have you considered defining an additional component to contain a bunch of `QueryParam`s, instead of throwing them into a JS array/object?